Makefile

```diff
@@ -8,8 +9,18 @@ TAG:=$(USER)$(shell date +%y%m%d_%H%M%S)
 
 DOCKER_TAG = $(DOCKER_PREFIX)$(IMAGE):$(TAG)
 
-# Pushes the combo image and the 3 smaller images
-all: docker-version docker-push-internal
+# Local targets:
+install: test
+	go install ./...
+
+test:
+	go test -timeout 30s -race ./...
+
+lint:
+	gometalinter ./...
```

Comment:
Running `make lint` exits with an error while all I see are warnings:
> make: *** [lint] Error 1

Do you also get the same result?

Reply:
you probably need to run:
```
 go get -u gopkg.in/alecthomas/gometalinter.v1
```
let me add that to the file

Reply:
or maybe you meant there are some errors, yes:
```
log/logger.go:166:6:warning: func name will be used as log.LogVf by other packages, and that stutters; consider calling this Vf (golint)
log/logger.go:196:6:warning: func name will be used as log.LogDebug by other packages, and that stutters; consider calling this Debug (golint)
log/logger.go:201:6:warning: func name will be used as log.LogVerbose by other packages, and that stutters; consider calling this Verbose (golint)
periodic/periodic.go:93:6:warning: type name will be used as periodic.PeriodicRunner by other packages, and that stutters; consider calling this Runner (golint)
```